src/main/java/org/takes/rs/RsWithCookie.java

```diff
@@ -90,6 +95,18 @@
     );
 
     /**
+     * Pattern for the expires cookie attribute.
+     */
+    private static final SimpleDateFormat EXPIRES_FORMAT = new SimpleDateFormat(
+        "'Expires='EEE, dd MMM yyyy HH:mm:ss 'GMT'",
```

Comment:
@danff I think  it would better to use `'Expires='EEE, dd MMM yyyy HH:mm:ss z` instead